### PR TITLE
Add tests for is-in-viewport.js

### PR DESCRIPTION
**Pull request by toolness**
This addresses part of #2046 but doesn't resolve all of it.

### CODE REVIEW

File: src/js/utils/is-in-viewport.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
 // https://stackoverflow.com/a/7557433
-module.exports = function isElementInViewport (el) {
+function isElementInViewport (el, win=window,
+                              htmlEl=document.documentElement) {
```
Comment (msecret) [inline src/js/utils/is-in-viewport.js]: When I see `htmlEl`, I don't really differentiate it from `el`. Should it be `docEl` or something?

Comment (msecret) [inline src/js/utils/is-in-viewport.js]: Would it make sense to have `win` and `htmlEl` be optional, defaulting to `window` and `documentElement` if they don't exist?

Comment (toolness) [inline src/js/utils/is-in-viewport.js]: Good idea, I will switch it to `docEl`... For your second comment, I think that is what I'm doing by using [ES6 default parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters)?

Comment (msecret) [inline src/js/utils/is-in-viewport.js]: Ha oh duh, I don't know why I didn't read it as that.

**Pull request merged**